command-expansion-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -41,35 +41,26 @@ export class Command<A extends ArgType[] | { [argName: string]: any } = [] | {}>
 
   public toSeqJson() {
     return {
-      id: 'command',
+      args: typeof this.arguments == 'object' ? Object.values(this.arguments) : this.arguments,
+      stem: this.stem,
+      time: {
+        tag: this.absoluteTime
+          ? this.absoluteTime.toString()
+          : this.relativeTime
+          ? this.relativeTime.toString()
+          : this.epochTime
+          ? this.epochTime.time.toString()
+          : '',
+        type: this.absoluteTime
+          ? 'ABSOLUTE'
+          : this.relativeTime
+          ? 'COMMAND_RELATIVE'
+          : this.epochTime
+          ? 'EPOCH_RELATIVE'
+          : 'COMMAND_COMPLETE',
+      },
+      type: 'command',
       metadata: {},
-
-      steps: [
-        {
-          stem: this.stem,
-          time: {
-            tag: this.absoluteTime
-              ? this.absoluteTime.toString()
-              : this.relativeTime
-              ? this.relativeTime.toString()
-              : this.epochTime
-              ? this.epochTime.time.toString()
-              : '',
-            type: this.absoluteTime
-              ? 'ABSOLUTE'
-              : this.relativeTime
-              ? 'COMMAND_RELATIVE'
-              : this.epochTime
-              ? 'EPOCH_RELATIVE'
-              : 'COMMAND_COMPLETE',
-          },
-          type: 'command',
-          metadata: {},
-          // include epoch field if we have an epoch time
-          ...(this.epochTime !== null ? { epoch: this.epochTime?.epochName } : {}),
```

Comment:
Looks like we removed epoch here?

Reply:
Yes that is correct. Below is the conversation with Mike and Taifun.

```
[@Michael L Schaffer](https://jpl.slack.com/team/U0BMC2CDP) circling back to epochs. If you are defining the epoch at the sequence activation, does that mean you can’t have a sequence use multiple epochs?
For my knowledge what is an epoch, do you have documentation about it?

[Taifun L O'Reilly](https://app.slack.com/team/U1M3ZU3HB)  [11 minutes ago](https://jpl.slack.com/archives/C015FPVM21Y/p1651683066033979?thread_ts=1651613519.197649&cid=C015FPVM21Y)
[@Ryan R Goetz](https://jpl.slack.com/team/U03RPSAQT) an epoch is an alias/name for a given time. A sequence can only refer to a single epoch. The 'e' time relation in the sequence refers to the value of the epoch the sequence is activated with. Take a look at the SEQ_ACTIVATE command.




[Michael L Schaffer](https://app.slack.com/team/U0BMC2CDP)  [10 minutes ago](https://jpl.slack.com/archives/C015FPVM21Y/p1651683129716039?thread_ts=1651613519.197649&cid=C015FPVM21Y)
[@Ryan R Goetz](https://jpl.slack.com/team/U03RPSAQT) correct you cannot reference multiple epochs in the same seq
```